tests/functional/search-non-auth.js

```diff
@@ -66,6 +66,35 @@ define([
         .end();
     },
 
+    'Clicking on label search adds query parameter to the URL': function() {
+      var params = '?q=dfjdkfjdkfjkdfjdkjf';
```

Comment:
I think we can remove `params` here and in the test below (as well as from `require.toUrl(url('/issues') + params)` -- it's not doing anything in these tests.